### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by alanngo**
learn shell

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -68,7 +68,7 @@ root@omarbelkady:~$ whoami
 ### echo
 Prints..... 
 ```bash
-root@omarbelkady:~$ echo "Nelan 56837 6342"
+root@omarbelkady:~$ echo "Omar 56837 74355"
```
Comment (usrbinomarbash) [inline README.md:71]: NO echo "63527 56837 6342!"

**Review by usrbinomarbash — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (alanngo) [reply README.md:71]: but you 56837 74355

Comment (usrbinomarbash) [reply README.md:71]: HELL NO

Comment (alanngo) [reply README.md:71]: echo "chromebooks suck"

Comment (usrbinomarbash) [reply README.md:71]: echo "56837 47 2 2-56837"

**Pull request closed without merging**